### PR TITLE
S4t eraser

### PR DESCRIPTION
**Pull request by georginaw**
Created eraser function which allows user to erase paint strokes from the canvas

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -1,30 +1,40 @@
...
+let paintColor = '#000000' // default paint color
 
-document.querySelectorAll('.row .pixel').forEach(function(pixel) {
-    pixel.style.backgroundColor = canvasColor
-})
+setCanvasColor(canvasColor)
 
-document.querySelector('.canvas').style.backgroundColor = canvasColor
+
+document.querySelector('nav .eraser').addEventListener('click', function () {
```
Comment (Incorrect-password) [inline script.js:10]: no space after function before ()

File: script.js [hunk 2]
```diff
@@ -1,30 +1,40 @@
...
 
-    let canvas =  canvasRow.repeat(row) // canvas (new variable) has a value of the row with 120 pixel divs BUT this itself is repeated 60 times)
+    let canvas =  canvasRow.repeat(row)
 
     document.querySelector('.canvas').innerHTML = canvas
 
 }
 
+function setCanvasColor(color) {
+    document.querySelectorAll('.row .pixel').forEach(function (pixel) {
```
Comment (Incorrect-password) [inline script.js:28]: no space

File: script.js [hunk 3]
```diff
@@ -1,30 +1,40 @@
...
+    })
+    document.querySelector('.canvas').style.backgroundColor = color
+}
+
+
+let clickdown
+
+let painting
 
 document.querySelector('html').addEventListener('mousedown', function () {
```
Comment (Incorrect-password) [inline script.js]: no space

File: script.js [hunk 4]
```diff
@@ -44,25 +54,24 @@ document.querySelector('.canvas').addEventListener('mouseenter', function () {
...
 
-document.querySelectorAll('.pixel').forEach(pixel => {
-    pixel.addEventListener('mousedown', function() {
-        painting = true
-        this.style.backgroundColor = paintColor
-    })
-    pixel.addEventListener('mousemove', function() {
-        if (painting === true && clickdown === true) {
+document.querySelectorAll('.row .pixel').forEach(pixel => {
+    setTimeout(function () {
```
Comment (Incorrect-password) [inline script.js]: no space

File: script.js [hunk 5]
```diff
@@ -44,25 +54,24 @@ document.querySelector('.canvas').addEventListener('mouseenter', function () {
...
+        pixel.addEventListener('mousedown', function () {
+            painting = true
             this.style.backgroundColor = paintColor
-        }
-    })
-    pixel.addEventListener('mouseup', function() {
-        painting = false
+        })
+        pixel.addEventListener('mousemove', function () {
```
Comment (Incorrect-password) [inline script.js]: no space

File: script.js [hunk 6]
```diff
@@ -44,25 +54,24 @@ document.querySelector('.canvas').addEventListener('mouseenter', function () {
...
+            if (painting === true && clickdown === true) {
+                this.style.backgroundColor = paintColor
+            }
+        })
+        pixel.addEventListener('mouseup', function () {
```
Comment (Incorrect-password) [inline script.js]: no space

File: script.js [hunk 7]
```diff
@@ -1,8 +1,12 @@
 
 generateCanvas(60, 100)
+
+
+
 let clickdown
```
Comment (Incorrect-password) [inline script.js]: let clickdown, painting will create a variable for both

**Review by Incorrect-password — CHANGES_REQUESTED** [review]

**Review by Noctiscreo — APPROVED** [review]
No changes needed.

**Pull request merged**